### ISSUE TITLE
list_of_keys.txt file should be removed from inputs folder once #544 is approved

### ISSUE THREAD

**Issue by ew3361zh** [RuminantFarmSystems/RuFaS#547]
In #541 it was helpful to add a txt file containing a list of variables which prevented the simulation from crashing and also showed the functionality of the newly created save_variables function. 

## Expected Behavior
This file is expected to be a holdover until #544 is ready to be merged in which should address the ability to resolve this issue.

## Current Behavior
Currently the file is present to support the functionality of #541.

## Possible Solution
Future changes (already in development) will move the save variables function to search in a directory to run for multiple files and if it doesn't find any, it will return an empty list (or dictionary) and therefore it won't crash the system if there aren't any present.

This file should, at that point, no longer be necessary and be able to be deleted both from the inputs folder and from the .gitignore file.

**Comment by ew3361zh** [comment]
no longer relevant
